src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -88,7 +88,7 @@
 
 {# Show link to import file sample #}
 {% macro import_file_sample(label, filename) %}
-    <a id="download-sample-{{ filename }}-file-link" class="list-group-item list-group-item-action" 
+    <a id="download-sample-{{ filename }}-file-link" class="list-group-item list-group-item-action"
```

Comment:
great enhancement ;-)

Reply:
Yeah, this one was added by @JevgenijVisockij I admit I'm not sure why 😅. But I guess the condition was maybe too loose previously and it generated an error with the latest version of Symfony and/or PHP